styles/burger.css

```diff
@@ -0,0 +1,183 @@
+/* Burger menu*/
```

Comment:
I wonder if the name of the file couldn't be changed to `menu.css` as it now contains almost all the styles associated with menu.